config.toml

```diff
@@ -0,0 +1,7 @@
+db = "freedit.db"
+addr = "127.0.0.1:3001"
+avatars_path = "static/imgs/avatars"
+inn_icons_path = "static/imgs/inn_icons"
+upload_path = "static/imgs/upload"
+tantivy_path = "tantivy"
+proxy = ""
```

Comment:
This file should be in the `apps/server` dir, unless it is now needed by the client?

Reply:
that file is even supposed to be .gitignored.  
I hadn't noticed all the default dirs and config stuff that get created by `config.rs`. So even though I moved the code in `apps/server`, all that config stuff (`config.toml`, `tantivy/`, `freedit.db` etc...) was still created in the monorepo root dir.
I refactored it in the last commit 
https://github.com/pseXperiments/freedit/pull/4/commits/bbf812cf931fd0b5ac32d94252e477c8c33f9a51#diff-fa36ea6251296f1ff94b66f84cbab988e9ed478733e2a611c3d458ea084ca940